pkg/state/release.go

```diff
@@ -117,7 +117,7 @@ func (r ReleaseSpec) ExecuteTemplateExpressions(renderer *tmpl.FileRenderer) (*R
 			}
 		}
 
-		result.Values = result.ValuesTemplate
+		result.Values = append(result.ValuesTemplate, result.Values...)
```

Comment:
This has some possibility to mutate result.Values.Template. I think we should use this intead:

```
result.Values = append([]string{}, result.Values...)
result.Values = append(result.Values, , result.ValuesTemplate...)`